### DISCUSSION TITLE
Laravel validation errors lost with polling

### DISCUSSION THREAD

**Discussion by elite123** [livewire/livewire#4977] [1. Help]
I have a form that is build using standard Laravel controller/view etc using the standard validator, with errors flashed to the session when validation fails.

When add a polling Livewire components to the page, this seems to clear the flash data if it polls after submit hit pressed, and there are no errors.

Removing `session()->forget(session()->get('_flash.new'));` from the following seems to prevent this happening

https://github.com/livewire/livewire/blob/3ce704ac3314fc8713f710ee1c4e865846569c19/src/Features/SupportRedirects.php#L45

Is there a way to turn this off or something I've missed?

Thanks

**Comment by joshhanley** [comment]
@elite123 Livewire works the same way as Laravel, in that data that is flashed to the session is removed on the next request. If you need to keep your flash messages around longer, I'd consider retrieving them from the session and either storing them in a property on the component, or manually adding them to the session, as it's only flashed messages that are removed (as per Laravel).

Hope this helps!
